save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/service/TestDiscoveringService.kt

```diff
@@ -97,17 +106,47 @@ class TestDiscoveringService {
             plugins.asSequence().flatMap { plugin ->
                 plugin.discoverTestFiles(testConfig.directory)
                     .map {
-                        val testRelativePath = it.test.toFile()
-                            .relativeTo(rootTestConfig.directory.toFile())
-                            .path
-                        TestDto(testRelativePath, plugin::class.simpleName!!, testSuite.id!!, it.test.toFile().toHash(), generalConfig.tags!!)
+                        val additionalFiles = if (plugin is FixPlugin) {
```

Comment:
what's about `FixAndWarnPlugin`?

Reply:
Ok, seems like `FixAndWarnPlugin#discoverTestFiles` returns us a `Sequence<FixTestFiles>`
Will fix it